build.sbt

```diff
@@ -28,3 +28,11 @@ libraryDependencies ++= {
 Revolver.settings
 
 javaOptions in Revolver.reStart += "-Dconfig.file=application.conf"
+
+// Copy over various properties
```

Comment:
I'm pretty sure that since restart fires up a forked jvm, it needs to know
about the config when doing so.
On Mar 31, 2015 4:33 PM, "David An" <notifications@github.com> wrote:

> In build.sbt
> <https://github.com/broadinstitute/vault-api/pull/35#discussion_r27520099>
> :
>
> > @@ -28,3 +28,11 @@ libraryDependencies ++= {
> >  Revolver.settings
> >
> >  javaOptions in Revolver.reStart += "-Dconfig.file=application.conf"
> > +
> > +// Copy over various properties
>
> I may be dense, but why is this necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/vault-api/pull/35/files#r27520099>.
>